examples/graphene_apps/python_worker/ov_workload/ov_workload_config.toml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ZMQ URL to send openvino work order requests.
+# There is a bug in GSC tool which cannot resolve DNS container hostname
+# to IP address.SO we need to assignstatic IP to conatiner.
```

Comment:
Change conatiner to container.

Reply:
ok

Reply:
And change assignstatic to assign static